app/View/Composers/MainMenuComposer.php

```diff
@@ -163,6 +161,9 @@ private function addMembersMenu(Builder $menu)
                              ->active('elections/*');
                     $miscMenu->add('Awards', route('award.season.index'))
                              ->active('awards/*');
+                    $miscMenu->add('PC deployment portal', config('bts.links.pc_deployment'));
+                    $miscMenu->add('Network management portal', config('bts.links.network_management'));
+                    $miscMenu->add('Telephony Portal', config('bts.links.telephony'));
```

Comment:
```suggestion
                    $miscMenu->add('Telephony portal', config('bts.links.telephony'));
```

just to be consistent